src/app/components/iterations-modal/iterations-modal.component.html

```diff
@@ -49,14 +49,14 @@
       </div>
       <div *ngIf="modalType=='create' || modalType=='update' || modalType=='start' || modalType=='createChild'"
         class="form-group datepicker-container">
-        <div class="date-container padding-right-5">
+        <div class="date-container padding-right-5 dib">
```

Comment:
what not just add `display: inline-block;` to the `.date-container` class, rather than add another class for the processor to read through?